### ISSUE TITLE
On macOS, could not find compatible Monarch UI for Flutter 3.16.0

### ISSUE THREAD

**Issue by fertrig** [Dropsource/monarch#136]
On macOS, after upgrading to Flutter 3.16.0 and running `monarch run` you get this message:

```
Downloading the Monarch UI for this project's flutter version...

We could not find a compatible Monarch UI for this version of Monarch and Flutter.
You could try switching Flutter versions. Or, create a GitHub issue in the Monarch repo 
and let us know which Flutter version your project is using: 
https://github.com/Dropsource/monarch/issues
```

Monarch is not yet compatible with Flutter 3.16.x due to a bug in the Flutter tools. This bug is fixed on the Flutter master branch. However, the fix hasn't made it to the stable branch yet. Here is the PR to cherry pick the fix to Flutter 3.16: https://github.com/flutter/flutter/pull/138518

**Comment by philipmjohnson** [comment]
I am also awaiting a 3.16.0 compatible Monarch. Thank you!

**Comment by enzoftware** [comment]
Also waiting for it.

**Comment by fertrig** [comment]
We are still awaiting the Flutter team to work on https://github.com/flutter/flutter/issues/138855 so Monarch can support Flutter 3.16.x on macos.

As a workaround, if you can use Flutter beta on your project then Monarch works with Flutter 3.18.x and 3.17.x.

**Comment by philipmjohnson** [comment]
I can confirm that Monarch now runs on MacOS on the recently released Flutter 3.19.0.  Thanks!
